devstats/Dockerfile.devstats

```diff
@@ -0,0 +1,38 @@
+# Dockerfile for devstats
+FROM golang:1.8
+
+RUN go get -u github.com/golang/lint/golint && \
+	go get golang.org/x/tools/cmd/goimports && \
+	go get github.com/jgautheron/goconst/cmd/goconst && \
+	go get github.com/jgautheron/usedexports && \
+	go get github.com/kisielk/errcheck && \
+	go get github.com/influxdata/influxdb/client/v2 && \
+	go get github.com/lib/pq && \
+	go get golang.org/x/text/transform && \
+	go get golang.org/x/text/unicode/norm && \
+	go get gopkg.in/yaml.v2 && \
+	go get github.com/google/go-github/github && \
+	go get golang.org/x/oauth2 && \
+	go get github.com/mattn/go-sqlite3
```

Comment:
BTW, @lukaszgryglicki Could we add vendor in devstats side in the future thus we could avoid `go get` in the docker image.

Reply:
@jlewi https://github.com/cncf/devstats/pull/99 devstas has vendor now and maybe we could not install so many deps in the docker image.